scripts/apps/authoring/authoring/directives/ArticleEditDirective.js

```diff
@@ -285,9 +285,23 @@ export function ArticleEditDirective(
                     scope.mediaLoading = true;
 
                     const defaultTab = crop ? 'crop' : 'view';
+                    let hideTabs = [];
 
-                    return renditions.crop(scope.item,
-                        {isNew: false, editable: true, isAssociated: false, defaultTab: defaultTab, showMetadata: true})
+                    if (scope.item.type !== 'picture') {
+                        hideTabs = ['image-edit', 'crop'];
+                    }
+
+                    return renditions.crop(
```

Comment:
There are multiple places where `renditions.crop` is called and I assume the rules which tabs to hide are generic. Wouldn't it be better to put the logic inside the `crop` method instead of using the same parameters in x places?

Reply:
You're right, pushing the fix now.